psi4/run_psi4.py

```diff
@@ -351,13 +351,18 @@ def custom_formatwarning(msg, *args, **kwargs):
 
     in_str = "Printing out the relevant lines from the Psithon --> Python processed input file:\n"
     lines = content.splitlines()
-    suspect_lineno = traceback.extract_tb(exc_traceback)[1].lineno - 1  # -1 for 0 indexing
-    first_line = max(0, suspect_lineno - 5)  # Try to show five lines back...
-    last_line = min(len(lines), suspect_lineno + 6)  # Try to show five lines forward
-    for lineno in range(first_line, last_line):
-        mark = "--> " if lineno == suspect_lineno else "    "
-        in_str += mark + lines[lineno] + "\n"
-    psi4.core.print_out(in_str)
+    try:
+        suspect_lineno = traceback.extract_tb(exc_traceback)[1].lineno - 1  # -1 for 0 indexing
+    except IndexError:
+        # module error where lineno useless (e.g., `print "asdf"`)
+        pass
+    else:
+        first_line = max(0, suspect_lineno - 5)  # Try to show five lines back...
+        last_line = min(len(lines), suspect_lineno + 6)  # Try to show five lines forward
+        for lineno in range(first_line, last_line):
+            mark = "--> " if lineno == suspect_lineno else "    "
+            in_str += mark + lines[lineno] + "\n"
+        psi4.core.print_out(in_str)
 
     # extact expection message and print it in a box for attention.
```

Comment:
Maybe you could fix the typos on this comment, while you're at it.

Reply:
👍 fixed 'em over at 2048